packages/@aws-cdk/aws-batch/test/test.batch.ts

```diff
@@ -1,8 +0,0 @@
-import { Test, testCase } from 'nodeunit';
-
-export = testCase({
-  notTested(test: Test) {
-    test.ok(true, 'No tests are specified for this package.');
-    test.done();
-  }
-});
```

Comment:
We need at least one integ test as well.

Reply:
I will write one up. Just a basic one for SPOT and ON_DEMAND. Will we need more?